src/ape_init/_cli.py

```diff
@@ -9,8 +9,26 @@
 
 @click.command(short_help="Initalize an ape project")
 @ape_cli_context()
-@click.option("--github")
+@click.option(
+    "--github", metavar="GH_username/repo", help="Add github_username/repository to clone project"
+)
 def cli(cli_ctx, github):
+    """
+    Ape Init allows the user to create an ape project with
+    default folders and ape-config.yaml
+
+        project_name/\n
```

Comment:
I think the extra \n are not needed, but otherwise this is fantastic!

Reply:
Docstrings via triple quotes render exactly as written, including all the newlines you have, so as Jules said no need for extra ones!